### DISCUSSION TITLE
Questions about Partitions

### DISCUSSION THREAD

**Discussion by tngok-wustl** [SkyrilHD/Dell-Exx50-Hackintosh#55] [Q&A]
Hi! I'm a university student take an iOS development course (which requires XCode on OS Monterey). I've searched for MacBook Pros online, but finally I've decided to build a Hackintosh. I've got a few questions:
1. The description under 'download & install' says 'Then, copy the EFI folder to your EFI partition...' What does exactly the 'EFI partition' here refer to? Is it Windows 10's EFI partition (if yes, how do I access that on my PC)?
2. I don't want to erase my original windows 10 OS on my laptop (I want to dual boot instead). Do you recommend me to create a partition for Hackintosh MacOS before installing the Hackintosh?

P.S. My laptop's specifications
Dell Latitude E7450
Windows 10 (x64)
x64-based processor
Intel(R) Core(TM) i5-5300U CPU @ 2.30GHz, 2295MHz, 2 Core(s), 4 Logical Processor(s)
16 GB DDR3 RAM (originally 8 GB, but I've recently installed another 8 GB RAM stick)
PNY CS900 500GB SSD
Intel(R) HD Graphics 5500

**Comment by SkyrilHD** [comment]
1. Any EFI partition, preferably the drive where you want to install macOS. A possible solution: [Link](https://www.insanelymac.com/forum/topic/311820-guide-mount-and-access-efi-partition-on-windows-10/)
2. You need to make sure your drive is formatted as GPT (GUID Partition Table), otherwise you will not be able to install macOS on the drive. Once you're in the macOS installer (assuming you have GPT), you can create a new partition in Disk Utility.app.

**Reply by tngok-wustl** [reply]
Vielen Dank für Ihre (oder deine) Antwort!

Now I've mapped my USB ports (generated a .kext file and a usb.json file). I've also followed this guide (https://dortania.github.io/OpenCore-Install-Guide/installer-guide/winblows-install.html#making-the-installer), formatted my USB drive using Rufus, copied my usb mapping .kext file to X64/EFI/OS/Kexts, copied the EFI folder from X64 to my USB drive and copied the 2 python downloaded files to com.apple.recovery.boot folder inside the USB drive. Am I doing the right thing?

Also, where can I disable XhciPortLimit?

**Reply by SkyrilHD** [reply]
Yes, the steps you mentioned are correct.

You can disable XhciPortLimit inside config.plist>Kernel>Quirks [Commit Link](https://github.com/SkyrilHD/Dell-Exx50-Hackintosh/commit/2454d4e61bd9e17d9ba3ce7b70c46786c3250003)

**Comment by tngok-wustl** [comment]
Thank you for your response!

Now I've downloaded your repo, copied the entire OC folder from the downloaded repo to my EFI (on my USB flash disk), and replaced your USB mapping kext with mine. I've also rebooted my laptop, and set my E7450 Bios settings according to the 'recommended BIOS settings' section in readMe.md (https://github.com/SkyrilHD/Dell-Exx50-Hackintosh/blob/master/README.md) - except for SATA operation, for which I am warned that **I will lose my windows 10 installation if I change the default (Raid On) SATA setting.** (I have also turned the 'Enable Legacy Option ROMs' on in the BIOS settings)

Then, I've rebooted my laptop again and try to boot from my usb stick. After booting, a MacOS-like interface pops out, with a disk icon in the middle (with the label 'Windows'); but after a few seconds, **the laptop automatically reboots one more time to my Windows OS.**

So, what's the problem? What other settings should I configure (without wiping out my Windows 10 OS)?

**Reply by SkyrilHD** [reply]
The UI with disk icon is OpenCore (a bootloader). By default, recovery installers are hidden and can be made visible by pressing the space bar. The laptop boots to Windows because the boot loader has a timeout of 5 seconds before it boots to the selected drive which is Windows.

Also, (I did not try this) you need to set your SATA setting to AHCI, otherwise macOS will not recognize your drive. You will not lose any data by changing the SATA setting. If Windows does not boot with AHCI, you can revert back to RAID.

**Comment by Skye10da** [comment]
> Hi! I'm a university student take an iOS development course (which requires XCode on OS Monterey). I've searched for MacBook Pros online, but finally I've decided to build a Hackintosh. I've got a few questions:
> 1. The description under 'download & install' says 'Then, copy the EFI folder to your EFI partition...' What does exactly the 'EFI partition' here refer to? Is it Windows 10's EFI partition (if yes, how do I access that on my PC)?
> 2. I don't want to erase my original windows 10 OS on my laptop (I want to dual boot instead). Do you recommend me to create a partition for Hackintosh MacOS before installing the Hackintosh?
> 
> P.S. My laptop's specifications
> Dell Latitude E7450
> Windows 10 (x64)
> x64-based processor
> Intel(R) Core(TM) i5-5300U CPU @ 2.30GHz, 2295MHz, 2 Core(s), 4 Logical Processor(s)
> 16 GB DDR3 RAM (originally 8 GB, but I've recently installed another 8 GB RAM stick)
> PNY CS900 500GB SSD
> Intel(R) HD Graphics 5500

How's your performance on pny cs900 SSD ?

**Comment by Skye10da** [comment]
> Hi! I'm a university student take an iOS development course (which requires XCode on OS Monterey). I've searched for MacBook Pros online, but finally I've decided to build a Hackintosh. I've got a few questions:
> 1. The description under 'download & install' says 'Then, copy the EFI folder to your EFI partition...' What does exactly the 'EFI partition' here refer to? Is it Windows 10's EFI partition (if yes, how do I access that on my PC)?
> 2. I don't want to erase my original windows 10 OS on my laptop (I want to dual boot instead). Do you recommend me to create a partition for Hackintosh MacOS before installing the Hackintosh?
> 
> P.S. My laptop's specifications
> Dell Latitude E7450
> Windows 10 (x64)
> x64-based processor
> Intel(R) Core(TM) i5-5300U CPU @ 2.30GHz, 2295MHz, 2 Core(s), 4 Logical Processor(s)
> 16 GB DDR3 RAM (originally 8 GB, but I've recently installed another 8 GB RAM stick)
> PNY CS900 500GB SSD
> Intel(R) HD Graphics 5500

How's your performance on pny cs900 SSD ?
